### PR TITLE
add sqlitecpp

### PR DESCRIPTION
**Pull request by wolfv**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/sqlitecpp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x' ) }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
```
Comment (scopatz) [inline recipes/sqlitecpp/meta.yaml]: doesn't this need ninja on windows?

Comment (wolfv) [reply recipes/sqlitecpp/meta.yaml]: nmake should be fine. However, windows seems to be still broken on staged-recipes at least.

**Comment by wolfv** [comment]
ready to go! skipping windows for now :)

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**